src/neptune_sacred/impl/__init__.py

```diff
@@ -17,47 +17,47 @@
 import json
 import os
 import warnings
+from typing import Union
 
+from neptune.new.handler import Handler
```

Comment:
Wouldn't this lead to user warning with 1.0 release?

Shouldn't we keep the `try` block?

Reply:
We should.